packages/dds/sequence/src/test/intervalCollection.events.spec.ts

```diff
@@ -259,7 +259,9 @@ describe("SharedString interval collection event spec", () => {
 					const [{ interval, previousInterval, previousEndpoints, local, op, slide }] =
 						eventLog;
 					assert.deepEqual(interval, { start: 0, end: 1 });
-					assert(toRemovalInfo(previousInterval.end.getSegment()) !== undefined);
+					const segment = previousInterval.end.getSegment();
+					assert(segment !== undefined);
+					assert(segmentIsRemoved(segment) === false);
```

Comment:
Is there a particular reason to not use `assertRemoved` for these cases?

Reply:
its not exported from the package, and i'd prefer it not be.

Reply:
not exporting it at all means it can be minimized, so keeps the bundle small, once it is exported, even internal, it can no longer be minimized

Reply:
makes sense!